test/parallel/test-http-import-websocket.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const {
+  WebSocket: NodeHttpWebSocket,
+  CloseEvent: NodeHttpCloseEvent,
+  MessageEvent: NodeHttpMessageEvent
+} = require('node:http');
+
+// Compare with global objects
+assert.strictEqual(NodeHttpWebSocket, WebSocket);
+assert.strictEqual(NodeHttpCloseEvent, CloseEvent);
+assert.strictEqual(NodeHttpMessageEvent, MessageEvent);
```

Comment:
*nitpick*

```suggestion
const http = require('node:http');

// Compare with global objects
assert.strictEqual(http.WebSocket, WebSocket);
assert.strictEqual(http.CloseEvent, CloseEvent);
assert.strictEqual(http.MessageEvent, MessageEvent);
```

IMO this makes more sense, as "NodeHttpXXX" feels like an excess for a name.

Reply:
I don't think is this is resolved.